autoload/ezguifont.vim

```diff
@@ -66,6 +66,10 @@ function! ezguifont#IncreaseFont()
   call s:adjust_font_size(g:ezguifont_step_size)
 endfunction
 
+function! ezguifont#ResetFontSize()
+  call s:set_font_size(g:ezguifont_default_font_size)
```

Comment:
It'd be nice to auto-detect the currently set font-size at plugin load time, then set that as default font size, instead of hardcoding it. However I'll leave this for a future improvement.